seqr/migrations/0052_rnaseqspliceoutlier_rank.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.18 on 2023-06-07 16:15
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('seqr', '0051_auto_20230426_1615'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='rnaseqspliceoutlier',
+            name='rank',
+            field=models.IntegerField(null=True),
```

Comment:
it looks like you changed the model definition but did not recreate the actual migration file

Reply:
Oh, you are right.